mlflow/server/js/scripts/global-setup.js

```diff
@@ -0,0 +1,25 @@
+const { execSync } = require('child_process');
+const os = require('os');
+
+module.exports = () => {
+  // On windows, the timezone is not set with `TZ=GMT`. As a workaround, use `tzutil`.
+  // This approach is taken from https://www.npmjs.com/package/set-tz.
+  if (os.platform() === 'win32') {
+    const TZ = 'GMT Standard Time';
+    const previousTZ = execSync('tzutil /g').toString();
+    const cleanup = () => {
+      execSync(`tzutil /s "${previousTZ}"`);
+      console.log(`Restored timezone to ${previousTZ}`);
+    };
+    execSync(`tzutil /s "${TZ}"`);
+    console.warn(
+      `Changed timezone to ${TZ}. If process is killed, manually run: tzutil /s "${previousTZ}"`,
+    );
+    process.on('exit', cleanup);
+    process.on('SIGINT', () => {
+      process.exit(2);
+    });
+  } else {
+    process.env.TZ = 'GMT';
+  }
+};
```

Comment:
@marijncv I saw your comment: https://github.com/mlflow/mlflow/pull/4941#issuecomment-958001972. I'm curious what `os.platform()` returns on bash pre-installed in WSL? Is it possible to check that?

Reply:
@harupy, thanks for the PR! It returns `'linux'`, see screenshot below.
![image](https://user-images.githubusercontent.com/9256667/140315708-a61be6c4-df38-4138-8596-3e6ad5776207.png)

Reply:
@marijncv Just out of curiosity, does `TZ=GMT react-app-rewired ...` work on bash in WSL?

Reply:
@harupy it does, but only when using `npm run ...`. See screenshots below:
![image](https://user-images.githubusercontent.com/9256667/140317970-0f74933a-0ab8-4b76-bed8-1e9feb1cfe87.png)
![image](https://user-images.githubusercontent.com/9256667/140318006-ebc7ce99-ee11-45f5-8f5f-0d9896e5f63c.png)

In any case I will start developing on WSL from now. Was using "regular" windows cmd prompt and powershell because in WSL the development server was not picking up code changes and running tests was really slow. But it turned out it was a configuration issue on my end and not WSL itself.

Reply:
Got it!